loops.js

```diff
@@ -33,11 +40,37 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
-
+for (let i=0; i<=15; i++) {
+    if (i === 0) {
```

Comment:
Deze if-statement kunt je eventueel kleiner maken omdat "0 % 2" ook 0 geeft (en dus "even" zal geven).  De hele if-else statement ziet er dan zo uit: 

`if (i % 2 === 0) {
        console.log(i + " is even");
    } else {
        console.log(i + " is oneven");
    }`